cmake/utils/git_pre-commit.in

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 
-STYLE=$PWD/.clang-format
+gitcmd=$(git rev-parse --show-superproject-working-tree)
```

Comment:
@lobis this is the way to get the superproject working tree in a git repository.

Do you want to add the `reformat-files` scripts from #208 in the this `pre-commit` script?

Reply:
I have added this into https://github.com/rest-for-physics/framework/pull/208, I merged this branch into my branch in order to test it